src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -43,13 +43,13 @@ class TaskKiller @Inject() (
           val activeInstances = foundInstances.filter(_.isActive)
 
           if (wipe) {
-            val decommissioned = await(Future.sequence(foundInstances.map(i => instanceTracker.setGoal(i.instanceId, Goal.Decommissioned))))
```

Comment:
I thought that someone told me that we need to assign the results when using async/await. But I don't remember who that was and I did not find anything about that limitations in async/await docs... 🤔

Reply:
You don't need to assign it. We assigned it to try and remove one warning, but it just generates another one. These are Future[Done] instead of Future[Unit] (which is helpful! but not for warnings).

Reply:
It was me and I did see a bug not assigning it. Maybe async/await had a bug or it was just coincident. Could we have a loop for this just to be sure? I think we can wait a day.

Reply:
huh, wild. Yeah we can run a loop

Reply:
off it goes! https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/marathon-sandbox/job/marathon-loop-smattering-warnings/